### PR TITLE
Stats class and use case skeletons

### LINKED ISSUE #8: Feature 8: Multiplayer interactions

### PR DESCRIPTION
**Pull request by adimaggio2021**

### CODE REVIEW

File: src/main/java/mariogatchi/Statistics.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        this.maxLevel = startingValue;
+        this.hunger = startingValue;
+        this.energy = startingValue;
+        this.cleanliness = startingValue;
+        this.happiness = startingValue;
+        this.agility = startingValue;
+        this.strategy = startingValue;
+        this.lastCheckTime = System.currentTimeMillis();
+    }
+
```
Comment (Najimsensei) [inline src/main/java/mariogatchi/Statistics.java:59]: I will be using this class on the Mariogatchi class

**Review by SpecialThing44 — APPROVED** [review]
Looks good and does what it needs to do, could maybe delete the HelloWorld classes.

**Review by Najimsensei — APPROVED** [review]
Looks good  Anthony!

**Review by Samuel-Glasroth — COMMENTED** [review]
Very good implementation, it does what it should do
In the constructor function you should have the if statement with curly braces due to convention
Also you should not have the magic number for starting numbers

**Review by Samuel-Glasroth — APPROVED** [review]
All good

**Comment by adimaggio2021** [comment]
This PR was accidentally made from the wrong branch. It was meant to be pulled in from the stats branch. The changed code contains the stats class, which in this iteration simply carries the mariogatchi's attributes, and the TimeRunner, which will be used to decrement stats as time goes by.

**Pull request merged**